### ISSUE TITLE
some emoji doesn't display properly

### ISSUE THREAD

**Issue by Jaehaks** [Allaman/emoji.nvim#34]
Thank you for your efforts, 

It works before but somedays, parts of emoji are not displayed properly like this

![image](https://github.com/user-attachments/assets/062e8f9f-2e9b-448f-9fe2-89cbd5126fed)

**Comment by Allaman** [comment]
Hello, thank you for your feedback. I think your terminal or font lack those icons.

This is how it looks for me with kitty:
<img width="502" alt="image" src="https://github.com/user-attachments/assets/53c0cc5a-ff2e-4786-9c06-075ad17de03d">

**Comment by Jaehaks** [comment]
@Allaman 

you are right, `windows-terminal-v1.21.2701.0` represents the emoji correctly. 
but `windows-terminal-preview-v1.22.2702.0` has a bug to show emoji. 
the emoji is inserted correctly, but it is not shown until i visualized the emoji character. 

Thanks
